### ISSUE TITLE
Failed to delete cookiefile: 'AlexaLogin' object has no attribute 'delete_cookiefile'

### ISSUE THREAD

**Issue by danielbrunt57** [alandtse/alexa_media_player#2470]
**Describe the bug**

Failed to delete cookiefile: 'AlexaLogin' object has no attribute 'delete_cookiefile'

**To Reproduce**

Update AMP to v4.12.8

**Expected behavior**

No error when deleting the config entry

**System details**

- Home-assistant (version): 
- - 2024.8.2
- alexa_media (version from `const.py` or HA startup):
- -  v4.12.8
- alexapy (version from `pip show alexapy` or HA startup): 
- - v1.28.2
- Amazon [2SV is enabled](https://github.com/alandtse/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n):
- -  Y

**Logs**

```
Source: custom_components/alexa_media/__init__.py:1339

Failed to delete cookiefile: 'AlexaLogin' object has no attribute 'delete_cookiefile'
```

**Solution (for now)**

Edit `/config/custom_components/alexa_media/__init__.py` and comment out lines 1335 - 1342

```
        # Delete cookiefile
#        try:
#            await login_obj.delete_cookiefile()
#            _LOGGER.debug("Deleted cookiefile")
#        except Exception as ex:
#            _LOGGER.error(
#                "Failed to delete cookiefile: %s",
#                ex,
#            )
```
![image](https://github.com/user-attachments/assets/a329505d-a513-469d-9636-f2a60819f87f)

**Comment by danielbrunt57** [comment]
New code in PR [#2471](https://github.com/alandtse/alexa_media_player/pull/2471):
```
        # Delete cookiefile
        if callable(getattr(AlexaLogin,'delete_cookiefile', None)):
          try:
              await login_obj.delete_cookiefile()
              _LOGGER.debug("Deleted cookiefile")
          except Exception as ex:
              _LOGGER.error(
                  "Failed to delete cookiefile: %s",
                  ex,
              )
        else:
            _LOGGER.warn(
                "Could not remove cookiefile: /config/.storage/alexa_media.%s.pickle."
                "  Please manually delete the file.",
                email,
            )
```

**Comment by danielbrunt57** [comment]
Fixed in AMP v4.12.9
